src/config/mod.rs

```diff
@@ -53,14 +54,13 @@ pub struct Config {
 }
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(default)]
 pub struct GlobalOptions {
     #[serde(default = "default_data_dir")]
     pub data_dir: Option<PathBuf>,
-    #[serde(
-        skip_serializing_if = "crate::serde::skip_serializing_if_default",
-        default
-    )]
+    #[serde(skip_serializing_if = "crate::serde::skip_serializing_if_default")]
     pub log_schema: LogSchema,
+    pub timezone: TimeZone,
```

Comment:
Should we also skip serializing this if it is the default (local)? My opinion would be yes.

Reply:
That would make sense.